Added email link for removing space. Added obfuscation filters

pedro-q opened the pull request:
Emails and phone numbers now display with * in the space profile, solves #86 . There's a new link that leads to an e-mail account so users can request removing their space, email account can be edited in templates/snippets/delete-space.html solves #87

pedro-q committed:
Added email link for removing space. Added obfuscation filtes to emails and phone numbers in the space profile

pedro-q commented:
New commit fixes #85

Lyla-Fischer commented on application/management/commands/load_csv.py:
```diff
@@ -56,26 +57,56 @@ def handle(self, *args, **options):
+                # Gets the affiliation ManyToMany option 
+                affiliation = space.pop('network_affiliation', None)
+                affiliation_obj = None
+                print(affiliation)
```
You left some print statements in here.

Lyla-Fischer commented on application/management/commands/load_csv.py:
```diff
@@ -14,7 +15,7 @@ def add_arguments(self, parser):
         data_filename = options['data_file_csv']
 
-        reverse_country_list = {name:code for code, name in countries}
+        reverse_country_list = {name:code for code, name in countries} 
```
Please remove the white space at the end of the line (there are some extensions that will do this for you, in most code editing software)

Lyla-Fischer commented on application/management/commands/load_csv.py:
```diff
@@ -56,26 +57,56 @@ def handle(self, *args, **options):
+                        setattr(new_space, field, None)
                 try:
-                    space.save()
+                    new_space.save()
```
I think you want to put the save after you adjust the ManyToMany fields, or else they won't persist.

pedro-q replied:
I though the same, but the space id field needs to be created first so the relationship can exist. It seems that the relationship persists but I'm gonna add an additional save() just in case.

Lyla-Fischer commented on application/templates/snippets/delete-space.html:
```diff
@@ -0,0 +1,5 @@
+<div class="container"> 
+	<div class="email-us">
+		<h5><a href="mailto:example@example.com">Email us to delete your space</a></h5>
```
@annawb This should go to anasegroup@gmail.com. Also, maybe have the wording be "Email us to be removed from the database". It's just gentler language.

Lyla-Fischer commented on application/templatetags/app_filters.py:
```diff
@@ -0,0 +1,15 @@
+@register.filter
+def obfuscate_phone(value): # Only one argument.
+    f = "".join(['*' for a in value[:-2]])+value[-2:]
+    return f
```
Please add a  newline at the end

Lyla-Fischer commented on application/templatetags/app_filters.py:
```diff
@@ -0,0 +1,15 @@
+register = template.Library()
+
+@register.filter
+def obfuscate_email(value): # Only one argument.
```
maybe rename this to "hide"? "Obfuscate" means to make confusing, but the data is still there.

Lyla-Fischer commented on application/templates/space_profile.html:
```diff
@@ -60,10 +60,10 @@ <h2 id="wTitle">{{space.name}}</h2>
             <!--<p >Contact Info</p>-->
             {% if space.email %}
-            <a href= "mailto:{{space.email}}" id="tealtext"><i class="material-icons tiny">mail</i> {{space.email | default_if_none:""}}</a><br>
+            <a href= "mailto:#" id="tealtext"><i class="material-icons tiny">mail</i> {{space.email | obfuscate_email | default_if_none:""}}</a><br>
```
If you are removing the email from the mailto, I would remove the whole anchor tag. We can add it back in later, if we end up doing that.

Lyla-Fischer commented:
Can you squash your commits unto just the two logical commits you started with? It will make it easier to track the history of certain ideas and changes in the future.

pedro-q committed:
Fixed the CSV importer

Fixed styling and code suggested from the comments in the pull request #88

Sorry forgot to add the extra save in the csv importer

pedro-q committed:
Added contributors page to about submenu

pedro-q commented:
New commit solves #43

Lyla-Fischer commented on application/templates/static/contributors.html:
```diff
@@ -0,0 +1,152 @@
+				<li>Maylan VN <small><i>&lt;maylanvn  at  gmail.com&gt;</small></i></li>
+				<li>David García </li>
+			</ul>
+		</div>
```
Let's put the code contributors in a "Contibutors.txt" file in the code repo, rather than on the contributors page on the site (because that tends to be more corporate rather than person-to-person)

Lyla-Fischer commented:
I left one quick fix comment on the contributors file, and then it lgtm.

pedro-q committed:
Added CONTRIBUTORS.txt, removed contributors from contributors page

Lyla-Fischer commented:
Thanks @pedroquiroz !
